dir.props

```diff
@@ -47,10 +47,15 @@
 
   <!-- Common build tool properties -->
   <PropertyGroup>
-    <BuildToolsPackageName>Microsoft.DotNet.BuildTools.CoreCLR</BuildToolsPackageName>
-    <BuildToolsVersion>1.0.1-prerelease</BuildToolsVersion>
-    <BuildToolsInstallSempahore>$(ToolsDir)BuildTools.$(BuildToolsVersion).installed.semaphore</BuildToolsInstallSempahore>
-    <BuildToolsTargetInputs>$(MSBuildThisFileFullPath);$(MSBuildThisFileDirectory)build.proj</BuildToolsTargetInputs>
-    <BuildToolsTargetOutputs>$(BuildToolsInstallSempahore)</BuildToolsTargetOutputs>
+
+    <BuildToolsCoreCLRPackageName>Microsoft.DotNet.BuildTools.CoreCLR</BuildToolsCoreCLRPackageName>
+    <BuildToolsCoreCLRVersion>1.0.1-prerelease</BuildToolsCoreCLRVersion>
+    <BuildToolsCoreCLRInstallSempahore>$(ToolsDir)BuildTools.$(BuildToolsCoreCLRVersion).installed.semaphore</BuildToolsCoreCLRInstallSempahore>
+
+    <BuildToolsPackageName>Microsoft.DotNet.BuildTools</BuildToolsPackageName>
+    <BuildToolsVersion>1.0.25-prerelease-00007</BuildToolsVersion>
```

Comment:
I will update this to the latest version of buildtools.

I will also move these to the dir.props file. No reason they should be here.